venues/ICLR.cc/2018/Conference/python/refresh-bibtex.py

```diff
@@ -0,0 +1,41 @@
+'''
+
+Iterates through the list of ICLR 2018 submissions and updates the bibtex field
+based on the most recent version.
+
+'''
+
+import argparse
+import openreview
+import config
+
+def get_bibtex(note):
+    firstWord = note.content['title'].split(' ')[0].lower()
+
+    return '''@article{
+  anonymous2018''' + firstWord + ''',
+  title={''' + note.content['title'] + '''},
+  author={Anonymous},
+  journal={International Conference on Learning Representations},
+  year={2018}
+}'''
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+submissions = client.get_notes(invitation=config.BLIND_SUBMISSION)
+
+for n in submissions:
+    # a submission potentially has many references, but we want to change only
+    # the "prime" reference, which has an ID equal to its referent.
+    refs = client.get_revisions(referent=n.id)
```

Comment:
Why don't you add a new revision with the new value?

BTW what you are doing is exactly to update note.

Reply:
Good point!

Reply:
It just occurred to me that we might not want the bibtex changes to appear in the revision history. what do you think @melisabok ?

Reply:
But _bibtex is hidden in the UI, right?

Ok, you can do an update then.

Reply:
I remember that you said that you were going to do this in the Add/Revision process function, what happened?

Reply:
ah, maybe you are right, that _bibtex is hidden.

well, I didn't do it because I originally thought it wasn't necessary. But it would be better if we can do it. So I will implement that now.